CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/experimental/TestParametersPlugin.java

```diff
@@ -107,6 +109,7 @@
     @ServiceProvider(service = Plugin.class)
 })
 @Messages("TestParametersPlugin=Test Parameters")
+@PluginInfo(pluginType = PluginType.SELECTION, tags = {"DEVELOPER", "MODIFY"})
```

Comment:
plugintype may be better suited to UPDATE?

Reply:
thank you @aldebaran30701, fixed!